deploy/04_prometheus-k8s_role.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Hm, why do we need a Role that gives the same permissions as the ClusterRole above, but in a specific namespace? That's not redundant?

Reply:
This role is for the openshift-monitoring serviceaccount `prometheus-k8s`. It needs these perms, but it shouldn't be given globally for just one operator.

Reply:
Yeah, this needs to scoped to the namespace.

Reply:
Okay, I get it now. This bit allows read access for the monitoring service for scraping; the clusterrole above is bound to the service account for the operator itself.